miraclelinux/meta-debian-extended
# xorg-xserver: Update version from 1.20.4-1+deb10u5 to 1.20.4-1+deb10u6

# Purpose of pull request

This version fixes following CVEs.
- CVE-2022-3550
- CVE-2022-3551

DLA: DLA 3185-1

# Test
## How to test

1. build package
2. Check core-image-weston works

## Test result

Build succeeded.

```
$ bitbake xserver-xorg
Loading cache: 100% |##########################################################################################################################################################################################################| Time: 0:00:00
Loaded 2358 entries from dependency cache.
Parsing recipes: 100% |########################################################################################################################################################################################################| Time: 0:00:13
Parsing of 1340 .bb files complete (787 cached, 553 parsed). 2358 targets, 78 skipped, 6 masked, 0 errors.
NOTE: Resolving any missing task queue dependencies

Build Configuration:
BB_VERSION           = "1.42.0"
BUILD_SYS            = "x86_64-linux"
NATIVELSBSTRING      = "ubuntu-18.04"
TARGET_SYS           = "aarch64-emlinux-linux"
MACHINE              = "qemuarm64"
DISTRO               = "emlinux"
DISTRO_VERSION       = "2.6"
TUNE_FEATURES        = "aarch64 armv8a crc"
TARGET_FPU           = ""
meta                 
meta-yocto-bsp       = "warrior:d4b57c68b22027c2bedff335dee06af963e4f8a8"
meta-debian          = "HEAD:19fb75a3dea2939553d93fd00e5ccf73aba8da7f"
meta-debian-extended = "update-xorg-server:14a00b41ca5136d4d53efdd7c522733d1c96ae09"
meta-emlinux         = "HEAD:a9f254c22c76555d82aa55eeb43ac92471394a7d"

Initialising tasks: 100% |#####################################################################################################################################################################################################| Time: 0:00:00
Sstate summary: Wanted 144 Found 0 Missed 144 Current 432 (0% match, 75% complete)
NOTE: Executing SetScene Tasks
NOTE: Executing RunQueue Tasks
NOTE: Tasks Summary: Attempted 2570 tasks of which 2037 didn't need to be rerun and all succeeded.
```

core-image-weston is not broken.

![core-image-weston](https://user-images.githubusercontent.com/165052/201574505-de9d4b39-494b-47d0-8231-310ad7779fd6.png)
